sdk/internal/recording/recording.go

```diff
@@ -807,7 +807,8 @@ func IsLiveOnly(t *testing.T) bool {
 
 // GetVariables returns access to the variables stored by the test proxy for a specific test
```

Comment:
Why do have the "/<extra stuff>" as the key name for the test if we are going to trim it ?